lib/remont/config.rb

```diff
@@ -6,7 +6,7 @@ class Config
     attr_accessor :process_timestamp_attribute
 
     # @return [String]
-    attr_accessor :script_path
+    attr_reader :script_path
```

Comment:
with rake task `:script_path` argument, I'd remove `script_path` config option

Reply:
You wouldn't have the default script path at all?

Reply:
exactly